content/en/tracing/setup/php.md

```diff
@@ -251,7 +252,7 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 | `DD_TRACE_DEBUG`                          | `false`     | Enable [debug mode](#custom-url-to-resource-mapping) for the tracer                                                                            |
 | `DD_TRACE_ENABLED`                        | `true`      | Enable the tracer globally                                                                                                                     |
 | `DD_TRACE_GENERATE_ROOT_SPAN`             | `true`      | Automatically generate a top-level span; set to `false` in conjunction with `DD_TRACE_AUTO_FLUSH_ENABLED=1` to trace long-running processes    |
-| `DD_TRACE_GLOBAL_TAGS`                    | `null`      | Tags to be set on all spans, for example: `key1:value1,key2:value2`.                                                                                 |
+| `DD_TAGS`                                 | `null`      | Tags to be set on all spans, e.g.: `key1:value1,key2:value2`.                                                                                  |
```

Comment:
Tiny nit: All of the `e.g.`'s were converted to `for example` in #7620.

Reply:
+1, for example is to ensure appropriate translation :)